ml_engine/online_prediction/predict_test.py

```diff
@@ -20,11 +20,10 @@
 import predict
 
 MODEL = 'census'
-JSON_VERSION = 'v1json'
-EXAMPLES_VERSION = 'v1example'
+JSON_VERSION = 'v2json'
 PROJECT = 'python-docs-samples-tests'
 EXPECTED_OUTPUT = {
-    u'confidence': 0.7760371565818787,
+    u'confidence': 0.7760370969772339,
```

Comment:
Do you test the value of the confidence? I'm little bit concerned because api response may have different confidence time to time.

Reply:
I tested this, and I believe it should be stable for the specific trained model (since the model's weights don't change and the input is always the same).

I was a little surprised that it differed slightly from the confidence returned by the old model version, since I used the same artifacts (I think? I just created a new model version, `v2json`, in the UI, based on the artifacts from the Cloud Storage directory listed for `v1json`). However, `v1json` seems to have been deployed as a TensorFlow 1.0, Python 2.7 model, but I redeployed `v2json` as a TensorFlow 1.15, Python 3.7 model; perhaps the serving code has changed its arithmetic or rounding between these..

Reply:
Ok, sounds good.